src/components/asset-panel/selector.jsx

```diff
@@ -15,6 +15,7 @@ const Selector = props => {
         buttons,
         containerRef,
         dragType,
+        dragPayload,
```

Comment:
18:9  error  'dragPayload' is assigned a value but never used. Allowed unused vars must match /^_/  no-unused-vars
  18:9  error  'dragPayload' is missing in props validation                                           react/prop-types

Maybe, it isn't defined in props...